packages/db-migrations/databases/fxa/patches/patch-144-145.sql

```diff
@@ -0,0 +1,196 @@
+-- UPDATE dbMetadata SET value = '143' WHERE name = 'schema-patch-level';
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('144');
+
+ALTER TABLE fxa.accounts ADD COLUMN clientSalt VARCHAR(128);
```

Comment:
Lets add a comment block here with a quick description of what version 2 is or maybe link out to spec.

Reply:
Comment above.